frontend/ui2/src/demos/layout/sidebar.tsx

```
@@ -0,0 +1,207 @@
+        <>
+          <div className="py-2">
+            <div className="space-y-1">
+              <div className="space-y-3">
+                <div
+                  className="text-sm font-medium ml-4 mb-1 cursor-pointer hover:text-primary"
+                  onClick={() => {
+                    // Toggle all top-level groups to collapse
+                    const event = new CustomEvent('collapse-all-groups');
+                    document.dispatchEvent(event);
+                  }}
+                >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)